packaging/debian/build_debian.sh

```diff
@@ -27,11 +27,28 @@ build_root="${2:-$(mktemp -d)}"
 
 echo "Building $mode mode in $build_root"
 
+build_electron() {
```

Comment:
Could we call this function `install_electron_dependencies` or something like that? It seems the actual building part happens elsewhere.